### DISCUSSION TITLE
Can I use Bard API with other programming languages?

### DISCUSSION THREAD

**Discussion by MasalaMarsala** [dsdanielpark/Bard-API#110] [Q&A] [answered]
Is there any way to use Bard API with other programming languages, or is it limited to only Python?

**Comment by dsdanielpark** [comment, marked as answer]
You can check this page. [Translation to Another Programming Language](https://github.com/dsdanielpark/Bard-API/blob/main/README_DEV.md#translation-to-another-programming-language) in README.md

Main feature can be translated to other programming language.

**Reply by MasalaMarsala** [reply]
Thanks!
